.github/workflows/merlin.yml

```diff
@@ -369,7 +369,7 @@ jobs:
       LOCAL_REGISTRY_PORT: 12345
       LOCAL_REGISTRY: "dev.localhost"
       INGRESS_HOST: "127.0.0.1.nip.io"
-      MERLIN_CHART_VERSION: 0.10.16
+      MERLIN_CHART_VERSION: 0.10.19
```

Comment:
@deadlycoconuts between this PR and https://github.com/caraml-dev/merlin/pull/397, guess we may need to coordinate a bit.

Reply:
I'll probably merge after this one 😅 So yep, I'll create the config in my PR. (Thanks for the reminder!)